azurerm_function_app/variables.tf

```diff
@@ -15,15 +15,6 @@ variable "resource_name_prefix" {
   description = "The prefix used to name all resources created."
 }
 
-locals {
-  # Define resource names based on the following convention:  # {azurerm_resource_name_prefix}-RESOURCE_TYPE-{environment}
-
-  azurerm_functionapp_name = "${var.resource_name_prefix}-functions-${var.environment}"
-
-  azurerm_functionapp_storage_account_name = "${var.resource_name_prefix}funcstorage${var.environment}"
-  azurerm_app_service_plan_name            = "${var.resource_name_prefix}-app-${var.environment}"
-}
-
 # TF_VAR_ADB2C_TENANT_ID
 variable "ADB2C_TENANT_ID" {
```

Comment:
is there a reason for this to be upper case?

Reply:
It was inherited from the azure.tf file, but it's not a problem to change it